tools/ci_build/builds/release_linux.sh

```diff
@@ -43,6 +43,12 @@ python3 -m pip install --upgrade setuptools
 python3 --version
 python3 ./configure.py
 
+## Run bazel test command. Double test timeouts to avoid flakes.
+bazel test -c opt -k \
+    --jobs=auto --test_timeout 300,450,1200,3600 \
+    --test_output=errors --local_test_jobs=8 \
+    //tensorflow_addons/...
```

Comment:
```suggestion
    --crosstool_top=//build_deps/toolchains/gcc7_manylinux2010-nvcc-cuda10.1:toolchain \
    //tensorflow_addons/...
```

Reply:
Whats happening is the test is building the custom-ops without the devtoolset-7 toolchain. The tests pass locally, but when combined with tensorflow from pip we'll get the #987 issue.

Because the test builds the ops already the build will not compile them again using the correct toolchian

Reply:
Thanks for the explanation!